tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -515,3 +554,250 @@ def test_positive_virt_who_hypervisor_subscription_status(session):
+            status='Enabled'
+        )
+        assert module_stream[0]['Name'] == FAKE_2_CUSTOM_PACKAGE_NAME
+        assert module_stream[0]['Stream'] == stream_version
+        assert module_stream[0]['Status'] == "Enabled"
+
+        # Disable Module Stream
+        result = session.contenthost.execute_module_stream_action(
+            vm_module_streams.hostname,
+            action_type='Disable',
+            module_name=FAKE_2_CUSTOM_PACKAGE_NAME,
+            stream_version=stream_version
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
+        module_stream = session.contenthost.search_module_stream(
+            vm_module_streams.hostname,
+            FAKE_2_CUSTOM_PACKAGE_NAME,
+            status='Disabled'
+        )
+        assert module_stream[0]['Name'] == FAKE_2_CUSTOM_PACKAGE_NAME
+        assert module_stream[0]['Stream'] == stream_version
+        assert module_stream[0]['Status'] == "Disabled"
+
+        # Reset Module Stream
+        result = session.contenthost.execute_module_stream_action(
+            vm_module_streams.hostname,
+            action_type='Reset',
+            module_name=FAKE_2_CUSTOM_PACKAGE_NAME,
+            stream_version=stream_version
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
+        assert not session.contenthost.search_module_stream(
+            vm_module_streams.hostname,
+            FAKE_2_CUSTOM_PACKAGE_NAME,
+            status='Disabled'
+        )
+        module_stream = session.contenthost.search_module_stream(
+            vm_module_streams.hostname,
+            FAKE_2_CUSTOM_PACKAGE_NAME,
+            status='Unknown'
+        )
+        assert module_stream[0]['Name'] == FAKE_2_CUSTOM_PACKAGE_NAME
+        assert module_stream[0]['Stream'] == stream_version
+        assert module_stream[0]['Status'] == ""
+
+
+@tier3
+def test_module_streams_customize_action(session, vm_module_streams):
+    """Check remote execution for customized module action is working on content host.
+
+    :id: b139ea1f-380b-40a5-bb57-7530a52de18c
+
+    :expectedresults: Remote execution for module actions should be succeed.
+
+    :CaseLevel: System
+
+    :CaseImportance: Medium
+    """
+    with session:
+        stream_version = "5.21"
+        run_remote_command_on_content_host(
+            'dnf -y upload-profile',
+            vm_module_streams
+        )
+        customize_values = {
+                'template_content.action': 'info',
+        }
+        # Run customize action on module streams
+        result = session.contenthost.execute_module_stream_action(
+            vm_module_streams.hostname,
+            action_type='Install',
+            module_name=FAKE_2_CUSTOM_PACKAGE_NAME,
+            stream_version=stream_version,
+            customize=True,
+            customize_values=customize_values
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
```

Comment:
Shouldn't you also assert that this action actually worked, i.e. result is any different than without customize?

I mean - I don't know the feature, so maybe that's not possible, but I would expect this customize action to do something visible.